ci-operator/step-registry/rosa/synthetics/e2e/setup/rosa-synthetics-e2e-setup-commands.sh

```
@@ -0,0 +1,104 @@
+set -o nounset
+set -o errexit
+set -o pipefail
+
+log() { echo -e "\033[1m$(date "+%H:%M:%S") $*\033[0m" >&2; }
+
+NS="rhobs-e2e-${BUILD_ID}"
+log "Ephemeral namespace: ${NS}"
+log "Mock API image: ${MOCK_API_IMAGE}"
+
+# --- OCM login + backplane ---
+source /usr/local/cs-qe-credentials/ocm-tokens
```

Shellcheck (0.11.0)

[info] Not following: /usr/local/cs-qe-credentials/ocm-tokens was not specified as input (see shellcheck -x).

(SC1091)